core/src/test/scala/org/apache/spark/sql/delta/DeleteSQLSuite.scala

```diff
@@ -56,7 +56,7 @@ class DeleteSQLSuite extends DeleteSuiteBase  with DeltaSQLCommandTest {
       withTempView("v") {
         Seq((1, 1), (0, 3), (1, 5)).toDF("key", "value").write.format("delta").saveAsTable("tab")
```

Comment:
Can you make sure to add tests for the following scenarios

1. DELETE on partitioned tables
2. DELETE with SUBQUERY
3. full table DELETE etc.